Library/Homebrew/brew.sh

```diff
@@ -296,6 +296,13 @@ source "$HOMEBREW_LIBRARY/Homebrew/utils/analytics.sh"
 setup-analytics
 report-analytics-screenview-command
 
+# Let user know we're still updating Homebrew if brew update --preinstall
+# exceeds 3 seconds.
+update-preinstall-timer() {
+  sleep 3
+  echo 'Still updating Homebrew, please wait...' >&2
```

Comment:
That would sound like it just started "updating Homebrew", leaving the user to wonder what it was doing in the past three seconds, or five. But I'll change it back if you insist.